src/cloud-foundry-api-client.js

```diff
@@ -27,6 +28,31 @@ class CloudFoundryAPIClient {
       ));
   }
 
+  fetchDeployerStatuses() {
+    return {
+      containerDeployer: this.fetchDeployerStatus('federalist-deploy-user'),
+      circleDeployer: this.fetchDeployerStatus('ci-deploy-user'),
+    };
+  }
+
+  fetchDeployerStatus(serviceName) {
+    let createdAt = null;
+    const deployService = cfenv.getAppEnv().getServiceCreds(serviceName);
+    if (deployService) {
```

Comment:
Curious about what kind of expectations we expect the deploy service to not have a created at date?

Reply:
the cloud.gov service accounts are all created on a date.  i'm not sure I follow ... please expound?

Reply:
Ah, I see this is checking that the deploy service is present. Let me re-phrase, when do we expect the deploy service to not be present?